tools/devops/automation/build-cronjob.yml

```diff
@@ -175,29 +58,30 @@ schedules:
   always: true
 
 stages:
-- template: templates/main-stage.yml
-  parameters:
-    xcodeChannel: Stable
-    isPR: false
-    provisionatorChannel: ${{ parameters.provisionatorChannel }}
-    pool: ${{ parameters.pool }}
-    runTests: ${{ parameters.runTests }}
-    runDeviceTests: ${{ parameters.runDeviceTests }}
-    runOldMacOSTests: ${{ parameters.runOldMacOSTests }}
-    runWindowsIntegration: ${{ parameters.runWindowsIntegration }}
-    runGovernanceTests: ${{ parameters.runGovernanceTests }}
-    runSamples: ${{ parameters.runSamples }}
-    enableDotnet: ${{ parameters.enableDotnet }}
-    enableAPIDiff: ${{ parameters.enableAPIDiff }}
-    forceInsertion: ${{ parameters.forceInsertion }}
-    skipESRP: ${{ parameters.skipESRP }}
-    ${{ if ne(length(parameters.simTestsConfigurations), 0)}}:
-      simTestsConfigurations: ${{ parameters.simTestsConfigurations }}
-    deviceTestsConfigurations: ${{ parameters.deviceTestsConfigurations }}
-    macTestsConfigurations: ${{ parameters.macTestsConfigurations }}
-    azureStorage: ${{ variables['azureStorage'] }}
-    azureContainer: ${{ variables['azureContainer'] }}
-    signingSetupSteps:
-    - template: ./templates/sign-and-notarized/setup.yml
-      parameters:
-        isPR: false
+
+- ${{ if eq(parameters.runGovernanceTests, true) }}:
+  - stage: governance_checks
+    displayName: 'Governance Checks'
+    jobs:
+    - job: governance
+      displayName: 'Governance Checks'
+      pool:
+        vmImage: windows-latest
+      steps:
+      - template: templates/governance-checks.yml
+        parameters:
+          isPR: false
+          repositoryAlias: self
+          commit: HEAD
+
+    - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
```

Comment:
One last thing, if we set the cron to just happen for main, we don't need this, do we?

Reply:
Good point!